lib.ftd

```diff
@@ -4538,3 +4540,296 @@ width: fill-container
+border-width.px: 1
+border-color: $inherited.colors.custom.two
+border-radius.px: 10
+height.fixed.px: 43
+margin-top.px: 8
+$on-blur$: $common.popup-form-validate()
+
+-- ftd.text: $signup-form-popup.org-warning
+id: user-org-warning
+role: $inherited.types.copy-small
+color: $fonts.dc3545
+
+-- end: ftd.column
+
+-- end: ftd.row
+
+-- ftd.text: $signup-form-popup.email-label
+role: $inherited.types.source-code
+margin-top.px: 16
+
+-- ftd.text-input:
+id: user-email-input
+padding.px: 6
+width: fill-container
+background.solid: transparent
+border-width.px: 1
+border-color: $inherited.colors.custom.two
+border-radius.px: 10
+min-height.fixed.px: 43
+margin-top.px: 8
+$on-blur$: $common.popup-form-validate()
+
+-- ftd.text: $signup-form-popup.email-warning
+id: user-email-warning
+role: $inherited.types.copy-small
+color: $fonts.dc3545
+
+-- ftd.row:
+width: fill-container
+spacing: space-between
+wrap: true
+
+-- cta-button: $signup-form-popup.cta-text
+bg-color: $inherited.colors.custom.four
+padding-horizontal: 41
+padding-vertical: 11
+marginTop: 2
+text-color: $inherited.colors.cta-primary.text
+$on-click$: $common.popup-form-validate()
+
+-- ftd.text: 
+id: user-label-success
+role: $inherited.types.button-large
+color: $inherited.colors.cta-primary.base
+align-self: end
+width.fixed.percent: 50
+width.fixed.percent if { ftd.device == "mobile" }: 100
+text-align: end
+margin-top.px: 20
+
+Thanks for your interest...We will update you
+
+-- end: ftd.row
+
+-- end: ftd.column
+
+-- end: ftd.column
+
+-- end: signup-form-popup
+
+
+
+
+
+
+
+
+
+
+-- component drop-down-1:
+caption title: $selected-item
+boolean $show-options: false
+common.options list drop-down-options-list:
+
+-- ftd.column:
+width: fill-container
+$on-click-outside$: $ftd.set-bool($a = $drop-down-1.show-options, v = false)
+
+-- ftd.row:
+id: user-identity-input
+align-self: center
+padding.px: 6
+border-radius.px: 10
+margin-top.px: 8
+width: fill-container
+height.fixed.px: 43
+border-color: $inherited.colors.custom.two
+border-color if { drop-down-1.show-options }: blue
+border-width.px: 1
+border-width.px if { drop-down-1.show-options }: 2
+spacing: space-between
+$on-click$: $ftd.toggle($a = $drop-down-1.show-options)
+
+-- ftd.text: $selected-item
+role: $inherited.types.copy-regular
+color: $inherited.colors.custom.one
+padding-right.px: 52
+min-width.fixed.px: 112
+
+-- ftd.image:
+src: $assets.files.static.faq.down.svg
+width.fixed.px: 16
+height: auto
+align-self: center
+
+-- end: ftd.row
+
+-- drop-down-1-options: $drop-down-1.title
+drop-down-1-options-list: $drop-down-1.drop-down-1-options-list
+$show-options: $drop-down-1.show-options
+
+-- end: ftd.column
+
+-- end: drop-down-1
+
+
+
+
+-- component drop-down-options-1:
+caption title:
+common.options list drop-down-options-1-list:
+boolean $mouse-in: false
+boolean $show-options: false
+
+-- ftd.column:
+if: { drop-down-options-1.show-options }
+
+-- ftd.column:
+background.solid: grey
```

Comment:
```suggestion
background.solid: $font.grey
```
`-- ftd.color grey: grey`
Define color inside `fonts.ftd` and call it from there